internal/controller/prefectserver_controller.go

```diff
@@ -179,32 +182,34 @@ func (r *PrefectServerReconciler) reconcileMigrationJob(ctx context.Context, ser
 
 	foundMigrationJob := &batchv1.Job{}
 	err = r.Get(ctx, types.NamespacedName{Namespace: server.Namespace, Name: desiredMigrationJob.Name}, foundMigrationJob)
-	if errors.IsNotFound(err) {
+
+	switch {
+	case errors.IsNotFound(err):
```

Comment:
i like to use switch cases when the if/else branching gets > 3

Reply:
I forgot that in Go you can use full expressions in the switch cases :bow: